src/pages/all-about-carbon/the-team.mdx

```diff
@@ -0,0 +1,150 @@
+| Robin Cannon         | Program Manager, Carbon Design System                       |
+| Mike Abbink          | Distinguished Designer, Carbon Design System                |
+| Dawn Ahukanna        | Design Principal and Front-end Architect, Cloud & Cognitive |
+| Haidy Francis        | Design Director, Security                                   |
+| David Levinson       | Visual Design Lead, Cloud Data & AI                         |
+| Tessa Rhodes         | Senior Design Manager, Cloud Platform                       |
+| Jeannie Servaas      | Creative Lead, Carbon Design System                         |
+| Wonil Suh            | Design Lead, Digital Design System                          |
+
+## Working groups
+
+Working groups are subject matter experts with a particular focus area. They can
+be formed from scratch, or might comprise an existing guild or group. They may
+be permanent initiatives, or formed for specific projects with a committed scope
+of work.
+
+Proposals for the formation of working groups are submitted to the steering
+committee for approval.
+
+### Current working groups
+
+#### Developer facilitation
+
+The key responsibilities of this group are:
+
+- Highlight technical challenges and obstacles to adoption, and recommended
+  tooling and solutions.
+- Work to ensure technical alignment of components and patterns with the system
+  as a whole.
+- Identify and encourage high quality project maintainers and committers.
+
+Members to be confirmed.
+
+#### Data visualization
+
+The key responsibilities of this group are:
+
+- Prioritize chart and data visualization types for design attention.
+- Support teams in using the best chart implementations for their product needs.
+
+Members to be confirmed.
+
+## Board of directors
+
+The Carbon Design System Board of Directors is comprised of executive
+stakeholders. They oversee the direction of Carbon.
+
+### Responsibilities
+
+The key responsibilities of the Board are as follows:
+
+- Hold the steering committee, and Carbon as a whole, accountable for delivery
+  of design excellence, standards, guidance and business impact.
+- Provide the steering committee feedback on business needs, to facilitate a
+  Carbon roadmap that meets those needs.
+- Champion Carbon adoption and Carbon standards within their organization.
+- Where possible, commit to engaging design or development resources from within
+  their organization to help accomplish the Carbon roadmap.
+
+The Board is supported by the Steering Committee, whose role is to provide
+oversight and direction for Carbon and its community, and related projects.
+
+Members of the Board meet for one half day every quarter, at which they review a
+quarterly report from the Steering Committee. This report details the status of
+design quality, adoption, and contribution from the product teams/business units
+from the Carbon Design System.
+
+### Members
```

Comment:
Do we get a board seat? E.g. Robin as the chair?

Reply:
Hmmm. I dunno, actually. Depends how much separation we want from the implementation and the accountability. Probably worth more research here.

Reply:
If the board is voting on anything, we'd just want to make sure that the core team's voices are heard. Since we report through Phil, maybe that's good enough as Phil could represent us.

Reply:
What the board see’s is already pretty vetted. My role is that for the team so whatever goes up to the board is already getting heavy evaluation and vetting by our team leadership, the steering group, and myself as the design executive. I’m happy not on it but I could be me so we have a seat at the table.